quickstart/dotnet-core/Program.cs

```diff
@@ -207,102 +208,97 @@ public static async Task ForceReconnectAsync()
-            Console.WriteLine("\nCache command  : " + cacheCommand + " or StringSet()");
-            Console.WriteLine("Cache response : " + cache.StringSet("Message", "Hello! The cache is working from a .NET Core console app!").ToString());
+            Console.WriteLine($"\nCache command: {cacheCommand} or StringSet()");
+            bool stringSetResult = await BasicRetryAsync(async (db) => await db.StringSetAsync("Message", "Hello! The cache is working from a .NET Core console app!"));
+            Console.WriteLine($"Cache response: {stringSetResult}");
 
-            // Demonstrate "SET Message" executed as expected...
             cacheCommand = "GET Message";
-            Console.WriteLine("\nCache command  : " + cacheCommand + " or StringGet()");
-            Console.WriteLine("Cache response : " + cache.StringGet("Message").ToString());
-
-            // Get the client list, useful to see if connection list is growing...
-            // Note that this requires allowAdmin=true in the connection string
-            cacheCommand = "CLIENT LIST";
-            Console.WriteLine("\nCache command  : " + cacheCommand);
-            var endpoint = (System.Net.DnsEndPoint)(await GetEndPointsAsync())[0];
-            IServer server = await GetServerAsync(endpoint.Host, endpoint.Port);
-            ClientInfo[] clients = await server.ClientListAsync();
-
-            Console.WriteLine("Cache response :");
-            foreach (ClientInfo client in clients)
-            {
-                Console.WriteLine(client.Raw);
-            }
+            Console.WriteLine($"\nCache command: {cacheCommand} or StringGet()");
+            getMessageResult = await BasicRetryAsync(async (db) => await db.StringGetAsync("Message"));
+            Console.WriteLine($"Cache response: {getMessageResult}");
 
             // Store .NET object to cache
             Employee e007 = new Employee("007", "Davide Columbo", 100);
-            Console.WriteLine("Cache response from storing Employee .NET object : " +
-            cache.StringSet("e007", JsonConvert.SerializeObject(e007)));
+            stringSetResult = await BasicRetryAsync(async (db) => await db.StringSetAsync("e007", JsonConvert.SerializeObject(e007)));
+            Console.WriteLine($"\nCache response from storing Employee .NET object: {stringSetResult}");
+
 
             // Retrieve .NET object from cache
-            Employee e007FromCache = JsonConvert.DeserializeObject<Employee>(cache.StringGet("e007"));
-            Console.WriteLine("Deserialized Employee .NET object :\n");
+            getMessageResult = await BasicRetryAsync(async (db) => await db.StringGetAsync("e007"));
+            Employee e007FromCache = JsonConvert.DeserializeObject<Employee>(getMessageResult);
+            Console.WriteLine($"Deserialized Employee .NET object :\n");
             Console.WriteLine("\tEmployee.Name : " + e007FromCache.Name);
```

Comment:
>\t [](http://example.com/codeflow?start=31&length=2)

eliminate '\t'? Or replace with `ConsoleKey.Tab`

Reply:
I tried to replace with ConsoleKey.Tab but it inserted "Tab" into the string 😁 the more you know. Removed the \t since it's not necessary for readability